apps/website/src/components/LiveExample.astro

```diff
@@ -16,9 +16,21 @@ export type Props = {
    * @default true
    */
   truncate?: boolean;
+
+  /**
+   * whether the live example will have a visible code snippet
+   * @default true
+   */
+  hasDemoCode?: boolean;
+
+  /**
+   * use to set the max height if it needs to be taller than the default
+   * @default '400px'
+   */
+  maxHeight?: string;
```

Comment:
I don't think this needs to be a prop. You should be able to increase the max-height in the CSS for all demos to 500px.

Reply:
Updated